package.json

```diff
@@ -49,11 +49,11 @@
     "@fluent/bundle": "^0.17.1",
     "@fluent/langneg": "^0.6.2",
     "@fluent/react": "^0.15.0",
+    "@leeoniya/ufuzzy": "^1.0.8",
```

Comment:
Just wanted to call out that this is a relatively new ([9 months old](https://github.com/leeoniya/uFuzzy/commit/f920effc32934fd014a0f8b2d9a5533504d6a383)) and not-that-widely-used library with a single maintainer. It's also probably relatively low-risk and [not particularly big](https://bundlephobia.com/package/@leeoniya/ufuzzy@1.0.8), but calling out in case I missed a consideration.

Reply:
There are definitely a couple of libraries that we could use here. This is the one I found gave the best results for the location data we have without diving too deep into their customization options. I gave a couple a quick try without comprehensive documentation:

- FlexSearch: https://github.com/nextapps-de/flexsearch
- Fuse.js: https://fusejs.io/
- Lunr.js: https://github.com/olivernn/lunr.js
- match-sorter: https://github.com/kentcdodds/match-sorter

Happy to try and make one of the previous options or another one work for us.